build/package/Microsoft.DotNet.Cli.Installer.DEB.proj

```diff
@@ -83,7 +83,7 @@
   </Target>
 
   <Target Name="TestSdkDeb"
-      Condition=" '$(OSName)' == 'ubuntu' and '$(DebuildPresent)'  == 'true' "
+      Condition=" '$(OSName)' == 'ubuntu' and '$(DebuildPresent)'  == 'true' and '$(IsSharedFrameworkPreReleaseVersion)' != 'True' "
```

Comment:
Not sure I understand this.  Why are you disabling these tests when we are on pre-release runtimes?